recipes/rcat/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "RCAT" %}
+{% set version = "v1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jplind79/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c539ba7d0030119e47a0376859a3e670976a1aa4a6061f26480826dd4a113545
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

See previous comment:
```suggestion
    - python >=3
```